### PR TITLE
Support video assets

### PR DESCRIPTION
**Pull request by zhiln**
[sc-28873]

### CODE REVIEW

File: dist/config.js [hunk 1]
```diff
@@ -73,6 +73,9 @@ var _default = {
   },
   video: {
     "class": 'body-video'
+  },
+  'video-asset': {
+    "class": 'body-video-asset'
```
Comment (ignatospadov) [inline dist/config.js:78]: Im wondering if it makes sense now to do the following:

```
assets: {
   'video': 'body-video-asset',
   ...
}
```

File: src/resolvers/contentful/asset.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { image, pdf } from '.'
+import { image, pdf, videoAsset } from '.'
```
Comment (ignatospadov) [inline src/resolvers/contentful/asset.js:1]: lets either call this a `video` or create a new subdirectory called `assets` and import from there

File: dist/config.js (hunk 1 above)
Comment (zhiln) [reply dist/config.js]: I added `asset` subcategory in `config`, put `image`, `pdf` and `video` there

File: src/resolvers/contentful/asset.js (hunk 2 above)
Comment (zhiln) [reply src/resolvers/contentful/asset.js:1]: Curerntly we don't have any subdirectories

Comment (ignatospadov) [reply src/resolvers/contentful/asset.js:1]: i think it makes sense to create a subcategory for assets, can `image` and `pdf` go there as well?

Comment (zhiln) [reply src/resolvers/contentful/asset.js:1]: yeah it could. It also make sense to stay consistent and create subcategory for `embed`

Comment (zhiln) [reply src/resolvers/contentful/asset.js:1]: Actually, what value would it bring? we'll have two files with the same name `video`. Not sure if it helps to navigate in the codebase

Comment (zhiln) [reply src/resolvers/contentful/asset.js]: fixed

**Review by ignatospadov — APPROVED** [review]

**Pull request merged**